src/main/java/adris/altoclef/tasks/resources/CollectCobbledDeepslateTask.java

```diff
@@ -0,0 +1,52 @@
+package adris.altoclef.tasks.resources;
+
+import adris.altoclef.AltoClef;
+import adris.altoclef.tasks.ResourceTask;
+import adris.altoclef.tasksystem.Task;
+import adris.altoclef.util.MiningRequirement;
+import net.minecraft.block.Block;
+import net.minecraft.block.Blocks;
+import net.minecraft.item.Items;
+
+public class CollectCobbledDeepslateTask extends ResourceTask {
```

Comment:
It looks like this is referenced from `CollectCobblestoneTask`, but in reality both tasks should be replaced with

```java
mine("cobblestone", MiningRequirement.WOOD, new Block[]{Blocks.COBBLESTONE, Blocks.STONE}, Items.COBBLESTONE);
mine("cobbled_deepslate", MiningRequirement.WOOD, new Block[]{Blocks.COBBLED_DEEPSLATE, Blocks.DEEPSLATE}, Items.COBBLED_DEEPSLATE);
```

I'll merge for now, but will make an issue so this can be addressed.